src/document.h

```diff
@@ -1435,6 +1440,13 @@ struct Document {
                 }
                 return nullptr;
 
+            case A_MERGE:
+                if (selected.xs == 1 && selected.ys == 1) return nullptr;
+                Action(dc, A_COPYCT);

```

Comment:
Ok thanks!
I tried a lot of ways but it does not seem that the codebase is made for merging cells? :-)

Reply:
It's not the codebase, its the way grids work. If you have a selection in the middle of a grid (meaning there are possibly occupied cells outside of your selection), what does it mean to merge cells? That can't always work, or at least, it is complicated what should happen. Hence, the code to delete cells is complicated: it has to check for all sorts of configurations to see if it can remove rows, or columns, or neither. Merging would be even more complicated since it would need to do what delete does but always retain at least one cell.
Doing what your code does is probably reasonable, it would just need some refactoring to not have the downsides of multiple undos or clipboard use etc. It be difficult to get right.

Reply:
Hello Wouter
Thanks for your explanation. I see the strings attached to such change and I now have a stripped down version that just join the text from the different cells without doing the deletion of the empty cells after. While this is then just half-way it might be a simpler approach.

Reply:
That is simpler, but also not so useful.. there is already a function to copy merged text, so the user can simply just use that to do the operation manually, copy the text, delete, and insert.